nova_vm/src/ecmascript/execution/environments.rs

```diff
@@ -555,3 +555,29 @@ impl Environments {
             .expect("ObjectEnvironmentIndex pointed to a None")
     }
 }
+
+/// ### {9.4.3 GetThisEnvironment ( )}(https://tc39.es/ecma262/#sec-getthisenvironment)
```

Comment:
**nitpick:** This should be `[]` brackets instead of braces, right?

Reply:
Fixed.